CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 ## New features
 
-* Add `EthSendRawTransactionUntrusted` RPC method to be used for the gateway when accepting `EthSendRawTransaction` and `eth_sendRawTransaction`. Applies a tighter limit on the number of messages in the queue from a single sender and applies additional restrictions on nonce increments. ([filecoin-project/lotus#12431](https://github.com/filecoin-project/lotus/pull/12431))
+- Add `eth_getBlockReceipts` RPC method to fetch all the receipts for a given block. ([filecoin-project/lotus#12447](https://github.com/filecoin-project/lotus/pull/12447))
```

Comment:
check the changelog here, you've added a ton of lint-style changes in here - some of these additional changes are fine, but don't belong in this PR, maybe you have a markdown linter installed in your editor that's doing these? Keep the change limited to the single addition of an entry for the change you're making.